arch/arm/soc/st_stm32/stm32f1/Kconfig.defconfig.series

```diff
@@ -12,6 +12,13 @@ source "arch/arm/soc/st_stm32/stm32f1/Kconfig.defconfig.stm32f1*"
 config SOC_SERIES
 	default stm32f1
 
+if CLOCK_CONTROL
```

Comment:
Can we do:

select CLOCK_CONTROL_STM32_CUBE if CLOCK_CONTROL

in arch/arm/soc/st_stm32/Kconfig so we don't need to duplicate this everywhere